ray_on_golem/provider/node_provider.py

```diff
@@ -309,3 +320,27 @@ def _stop_webserver(ray_on_golem_client: RayOnGolemClient) -> None:
                 return
 
             cli_logger.print("Requesting webserver shutdown done, will stop soon")
+
+    def _print_mainnet_onboarding_message(self, yagna_payment_status_output: str) -> None:
+        if self._payment_network not in MAINNET_PAYMENT_NETWORKS:
+            return
+
+        cli_logger.newline()
+
+        with cli_logger.indented():
+            cli_logger.print(
+                "Running Ray on Golem on the mainnet requires GLM and MATIC tokens on the Polygon blockchain"
```

Comment:
well, we're checking for `mainnet` but then only seem to support Polygon ...

we should either:
* only print this message if the user is indeed using Polygon and print a different message for actual _mainnet_ users
* or ensure that our onboarding also supports the Ethereum Mainnet and differentiate the message depending on the actual blockchain used by the user